frontend/src/components/store/PluginCard.tsx

```diff
@@ -116,6 +116,9 @@ const PluginCard: FC<PluginCardProps> = ({ plugin }) => {
             <p className={joinClassNames(staticClasses.PanelSectionRow)}>
               <span>Author: {plugin.author}</span>
             </p>
+            <p className={joinClassNames(staticClasses.PanelSectionRow)}>
+              <span>Description: {plugin.description}</span>
```

Comment:
Don't really think a Description: prefix is necessary and it takes up more screen space.

Reply:
Honestly I agree, but since the other fields had prefixes I just followed convention.

Reply:
personally I think the description is a good idea but it should be faded after a certain character limit.

Reply:
@TrainDoctor that is not what I meant, I meant if the `Description:` prefix should be removed since it takes up a lot of space

Reply:
> @TrainDoctor that is not what I meant, I meant if the `Description:` prefix should be removed since it takes up a lot of space

A gotcha. Yeah no need for the prefix

Reply:
Prefix removed.